README.md

```diff
@@ -24,10 +24,20 @@ The restore service consists of a website that allows you to view your backups a
 
 ### Functions
 
-1. `cd functions`
-1. `python -m venv .venv`
+#### Python
+
+The firebase emulator requires that you create a virtual environment. It does not seem to recognize conda environments.
+
+1. `cd functions/python`
+1. `python -m venv venv`
```

Comment:
What version of python does this expect? 

This might be problematic since we can't choose a python version. 

A tool like uv would allow that.